sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTest.java

```diff
@@ -66,7 +66,12 @@ protected void createKeyClient(HttpClient httpClient, KeyServiceVersion serviceV
     public void createKey(HttpClient httpClient, KeyServiceVersion serviceVersion) {
         createKeyClient(httpClient, serviceVersion);
 
-        createKeyRunner((keyToCreate) -> assertKeyEquals(keyToCreate, keyClient.createKey(keyToCreate)));
+        createKeyRunner((keyToCreate) -> {
+            KeyVaultKey createdKey = keyClient.createKey(keyToCreate);
+
+            assertKeyEquals(keyToCreate, createdKey);
+            assertNotNull(createdKey.getProperties().getHsmPlatform());
```

Comment:
should we have a check for the actual value also ?

Reply:
See my other comment about this.